### PR TITLE
Uranium Ammo

### PR DESCRIPTION
**Pull request by Boaz1111**
## About the PR

I added uranium ammo, made practice ammo roundstart instead of locked behind a research and made draconic munitions research cheaper (6k)
## Why / Balance

https://github.com/space-wizards/space-station-14/issues/21462
## Technical details

Nuh uh
## Media

- [X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

nuh uh
**Changelog**

- add: Added uranium munitions, a new T1 arsenal research.
- tweak: Made practice munitions and weapons roundstart

### CODE REVIEW

**Comment by JoeHammad1844** [comment]
what does DU ammo do?

**Comment by JoeHammad1844** [comment]
the damage shouldnt mostly be radiation, it would be better if it was half and half with pierce

**Comment by JoeHammad1844** [comment]
and you should probably make it available to nukies somehow, otherwise its just more anti-nukie powercreep and will make warops basically impossible

**Comment by JoeHammad1844** [comment]
wait wait wait, this shit is TIER 1? that need to go to T2 or 3, that is too powerful to be tier fucking ONE

**Comment by Boaz1111** [comment]
> wait wait wait, this shit is TIER 1? that need to go to T2 or 3, that is too powerful to be tier fucking ONE

how? Incendiary is T1

**Comment by JoeHammad1844** [comment]
> how? Incendiary is T1

it deals radiation damage, which is not common for nukies to carry (infact i have NEVER seen it because the chems they can make are so limited) they do have a radiation medkit but its only the one and that isnt enough to treat the team if EVERY BULLET DEALS RADIATION

**Comment by Holinka4ever** [comment]
> > how? Incendiary is T1
> 
> it deals radiation damage, which is not common for nukies to carry (infact i have NEVER seen it because the chems they can make are so limited) they do have a radiation medkit but its only the one and that isnt enough to treat the team if EVERY BULLET DEALS RADIATION

If i remember well, nukie armor has radiation defence, 40% at least.

You also need to research them, due in hidden nuke is not possible because of armory breaking.
 
In case, you need actually to aim cause its a projectile, nor a laser hitscan.

We actually have laser-radiation weapon, but seems people are okay with it.

**Comment by JoeHammad1844** [comment]
nukies will technically take the same amount of damage as a regular bullet, but they dont commonly have the chemicals to heal radiation, and im sure everyone would be okay with Xray cannons being tier 3. and by having these be ballistic you get around the drawbacks of laser weapons, a low fire rate and ammo pool.

you can load up a WT550 with 30 of these and piss them down a hallway. even worse with a fucking L6 saw or a drozd

**Comment by Holinka4ever** [comment]
> nukies will technically take the same amount of damage as a regular bullet, but they dont commonly have the chemicals to heal radiation, and im sure everyone would be okay with Xray cannons being tier 3. 
If i remember well, xray cannons are in tier 2, in tier 3 there is only cap laser

> you can load up a WT550 with 30 of these and piss them down a hallway. even worse with a fucking L6 saw or a drozd
What about to kill them first? I think it will work fine!

**Comment by AjexRose** [comment]
Radiation based ammo seems like an easy win, at least with the x-ray cannon, you cant switch mags or easily swap to another one, and it can be easily deflected with a shield or e-sword. Hell, I don't even think a desword can reliably deflect regular bullets. 
After doing some quick googling, it seems that depleted uranium ammo isn't even radioactive enough to penetrate human skin, and seems to fit more as either a high velocity or armor piercing role.

File: Resources/Prototypes/Research/arsenal.yml
Comment (c0rigin) [inline Resources/Prototypes/Research/arsenal.yml:1]: Try to put unrelated changes in a different pull request, such as the incendiary ammunition research change.

Comment (JoeHammad1844) [reply Resources/Prototypes/Research/arsenal.yml:1]: this PR is about bullets, it is also about research
incendiary bullets are bullets and also research

**Comment by Boaz1111** [comment]
Renamed them to uranium instead of depleted uranium and nerfed them

**Comment by Minty642** [comment]
i expect uranium ammo to have better piercing or multi-piercing instead of radiation damage

**Comment by JacktheJukebox** [comment]
I kinda liked the depleted uranium idea if it worked as AP

gave sec armor piercing capability locked behind less damage and arsenal tech

**Comment by Boaz1111** [comment]
> I kinda liked the depleted uranium idea if it worked as AP
> 
> 
> 
> gave sec armor piercing capability locked behind less damage and arsenal tech

I'm not putting AP ammo in T1, that shit kills so fucking fast

**Comment by Boaz1111** [comment]
> i expect uranium ammo to have better piercing or multi-piercing instead of radiation damage

I changed them from depleted uranium to uranium, do you really need me to change it to like "uranium-tipped ammo"

File: Resources/Prototypes/Recipes/Lathes/security.yml [hunk 1]
```diff
@@ -455,3 +455,75 @@
...
+    Plastic: 20
+    Uranium: 2
+
+- type: latheRecipe
+  id: CartridgeRifleUranium
+  result: CartridgeRifleUranium
+  completetime: 2
+  materials:
+    Plastic: 15
+    Uranium: 1.5
```
Comment (DangerRevolution) [inline Resources/Prototypes/Recipes/Lathes/security.yml:497]: You can't have half a sheet of uranium, it's 1 or 2

Comment (Boaz1111) [reply Resources/Prototypes/Recipes/Lathes/security.yml:497]: It did!

File: Resources/Textures/Objects/Weapons/Guns/Ammunition/Casings/shotgun_shell.rsi/meta.json
Comment (ficcialfaint) [inline Resources/Textures/Objects/Weapons/Guns/Ammunition/Casings/shotgun_shell.rsi/meta.json:1]: Are the sprites from discordia-space/CEV-Eri as well? If not, you need to specify the place you took them from in `copyright`

File: Resources/Textures/Objects/Weapons/Guns/Projectiles/projectiles2.rsi/meta.json
Comment (ficcialfaint) [inline Resources/Textures/Objects/Weapons/Guns/Projectiles/projectiles2.rsi/meta.json:1]: See the comment above

**Review by Emisse — APPROVED** [review]
